index.html

```diff
@@ -30,9 +31,42 @@ <h1>FizzBuzz問題</h1>
   ptag.textContent = '【出力】';
   outputArea.appendChild(ptag);
 
-for (let i = 1; i < 100; i++) {
-    let value ='';
-if (i % fizzNumber === 0 && i % buzzNumber === 0) {
+  function NumberClear() {
+    elemFizzNumber.value = '';
+    elemBuzzNumber.value = '';
+    elemFizzNumber.focus();
+  }
+
+  function errorMessage() {
+   const checker = document.createElement('p');
+   checker.textContent = '整数値を入力してください';
+   const div = document.querySelector('div');
+   outputArea.appendChild(checker);
+   }
+
+   if (isNaN(fizzNumber) || isNaN(buzzNumber)) {
+     value = '整数値を入力してください';
+     NumberClear();
+     return errorMessage();
+     }
+
+   if ((fizzNumber == 0) || (buzzNumber == 0)) {
+    value = '整数値を入力してください';
+    NumberClear();
+    return errorMessage();
+    }
+
+   if (fizzNumber.match(/^-?[0-9]+\.[0-9]+$/) || buzzNumber.match(/^-?[0-9]+\.[0-9]+$/)) {
+    value = '整数値を入力してください';
+    NumberClear();
+    return errorMessage();
+    }
+
+const range = [...Array(100)].map((v,i) => i + 1);
+
+for　(i = 1; i < range.length; i++){
```

Comment:
for 文中の一時変数はきちんと `let` で宣言するようにしてください。
```suggestion
for　(let i = 1; i < range.length; i++){
```